### PR TITLE
remove strats

### PR DESCRIPTION
**Pull request by peterMangrove**
The purpose of this PR is to split Core and Strat contracts in separate repos. This is done to be very clear what is the core of Mangrove and it keeps Core and Strats code from being mixed up together.

This PR removes all strat related files (contracts, scripts, deployers, etc.). Some files needed small changes in order for them to work, with all the removed files.

A following PR in the new [mangrove-strats ](https://github.com/mangrovedao/mangrove-strats)repo will do the same, where all Core related files will be removed.

### CODE REVIEW

File: config.js [hunk 1]
```diff
@@ -6,20 +6,7 @@ exports.abi_exports = [
   "MgvOracle",
   "TestToken",
   "IERC20",
-  "MangroveOrder",
```
Comment (lnist) [inline config.js:9]: Clean up addresses/*?

File: test/toy_strategies/TestToken.t.sol
Comment (espendk) [inline test/toy_strategies/TestToken.t.sol:1]: Should this have been deleted?

**Review by espendk — DISMISSED** [review]
Nice!

Would be great if the PR described the motivation and steps for this change + pointed to the new strat repo.

We could consider keeping more deployers here, though it might require a bit of refac. Examples:
- MumbaiActivateMarket (cannot activate MangroveOrder)
- MumbaiFaucet* (could be kept unchanged?)
- ERC20Deployer and related

File: config.js (hunk 1 above)
Comment (peterMangrove) [reply config.js:9]: Done

File: test/toy_strategies/TestToken.t.sol
Comment (peterMangrove) [reply test/toy_strategies/TestToken.t.sol:1]: Reinstanted test, with removed unused imports

**Review by lnist — APPROVED** [review]

**Review by espendk — APPROVED** [review]
👌

**Pull request merged**